src/app/applications/datatarget/fiware/fiware-edit/fiware-edit.component.html

```diff
@@ -104,23 +125,20 @@ <h6> {{'QUESTION.DATATARGET.RELATIONS' | translate}} </h6>
                 <div class="row">
                   <mat-form-field appearance="fill">
                     <mat-label>{{'QUESTION.DATATARGET.SELECT-DEVICES' | translate}}</mat-label>
-                    <mat-select
-                      multiple
-                      name="devices"
-                      [compareWith]="compare"
-                      [(value)]="element.iotDeviceIds">
+                    <mat-select multiple
```

Comment:
Is this just re-formatted? If so, avoid formatting code unrelated to the actual feature. It makes it harder to tell what to focus on.
Generally, when formatting, follow the existing style or use the pre-configured formatter in the project to avoid conflicts. Unfortunately, there's no formatter configured for HTML files in this project.

Reply:
Yeah, that's probably the reason. Formatting in HTML is not consistent and probably the editor just changed it automatically. Now it is supposed to be reverted.